src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
 export interface Figure {
+  shape: string;
+  color: string;
 
+  getArea(): number;
 }
 
-export class Triangle {
+function round(expression: number): number {
+  return Math.floor(expression * 100) / 100;
+}
+
+enum ShapeFigure {
+  triangle = 'triangle',
+  circle = 'circle',
+  rectangle = 'rectangle'
+}
 
+enum ColorFigure {
+  red = 'red',
+  green = 'green',
+  blue = 'blue'
 }
 
-export class Circle {
+export class Triangle implements Figure {
+  constructor(
+    public color: string = ColorFigure.green,
```

Comment:
```suggestion
    public color:  ColorFigure,
```
`ColorFigure` is the type, no need to write `string` here
and it could be any color, not only green)